clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/ModifyGoogleServerGroupInstanceTemplateAtomicOperation.groovy

```diff
@@ -176,9 +176,9 @@ class ModifyGoogleServerGroupInstanceTemplateAtomicOperation implements AtomicOp
         instanceTemplateProperties.setTags(tags)
       }
 
-      // Override the instance template's auth scopes if authScopes was specified.
-      if (overriddenProperties.authScopes != null) {
-        def serviceAccount = GCEUtil.buildServiceAccount(description.authScopes)
+      // Override the instance template's service account if serviceAccountEmail or authScopes was specified.
+      if (overriddenProperties.serviceAccountEmail != null || overriddenProperties.authScopes != null) {
```

Comment:
Ahh - sounds like a perfect candidate for a comment on why you're explicitly comparing to `null`.